proto/waste.proto

```diff
@@ -0,0 +1,102 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+import "types/info.proto";
+import "types/change.proto";
+
+// WasteApi exposes details about units of waste that have been produced by the building
+service WasteApi {
+  rpc ListWasteRecords(ListWasteRecordsRequest) returns (ListWasteRecordsResponse);
+  rpc PullWasteRecords(PullWasteRecordsRequest) returns (stream PullWasteRecordsResponse);
+}
+
+service WasteInfo {
+  rpc DescribeWasteRecord(DescribeWasteRecordRequest) returns (WasteRecordSupport);
+}
+
+enum DisposalMethod {
+  DISPOSAL_UNSPECIFIED = 0;
```

Comment:
The UNSPECIFIED enum field should be spelled exactly as `toConstCase(enumName)+'_UNSPECIFIED'` - aka `DISPOSAL_METHOD_UNSPECIFIED`